.github/workflows/test.yml

```
@@ -0,0 +1,310 @@
+name: SalesforceCommerceCloud/pwa-kit/test
+on:
+  schedule:
+    - cron: 0 8 * * *
+  push:
+     branches:
+     # TODO: Before merging remove branches filter
+     - github-actions
+jobs:
+  pwa-kit:
+    strategy:
+      fail-fast: false
```

why do we set this to false? I think fail fast is very useful to have.

---

We need `fail-fast: false` because this allows all queued jobs in the matrix to run even if one fails! Without it, if one job fails, then GitHub will cancel all in-progress and queued jobs

---

Remove this line before merging. The line was added during development for debugging, in real builds we want to cancel all builds as soon as one fails. 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#:~:text=GitHub%20will%20cancel%20all%20in%2Dprogress%20and%20queued%20jobs%20in%20the%20matrix%20if%20any%20job%20in%20the%20matrix%20fails.%20This%20property%20defaults%20to%20true